### PR TITLE
Adds a floating action button with help and chat links to every page.…

### LINKED ISSUE #2124: Add configurable help link to every page
We might get better engagement from users by having a "chat" style link on each page that can be configured to point the user to appropriate chat room and help resources.

### PR DESCRIPTION
**Pull request by alexec**
… Closes #2124
Screen Shot 2019-08-16 at 15 59 54
Screen Shot 2019-08-15 at 14 10 40
* [ ] Can we collect analyitcs?

### CODE REVIEW

**Comment by alexec** [comment]
Fails on `TestHookDeleteBeforeCreation` - known flaky test.

File: ui/src/app/app.tsx [hunk 1]
```diff
@@ -165,6 +166,11 @@ export class App extends React.Component<{}, { popupProps: PopupProps, error: Er
                             <Redirect path='*' to='/'/>
                         </Switch>
                     </Router>
+                    <Container>
+                        <Link href='https://argoproj.github.io/argo-cd/' tooltip='Docs' icon='fas fa-book'/>
+                        <Link href='https://argoproj.slack.com/messages/CASHNF6MS' tooltip='Chat' icon='fas fa-comment-alt'/>
```
Comment (alexec) [inline ui/src/app/app.tsx]: I think this should be configurable

**Comment by jessesuen** [comment]
Can we change button to: "Get Help" or make it configurable?

**Comment by alexec** [comment]
@jessesuen this now has configurable help text, I've provided an example above.

**Comment by alexec** [comment]
I'd like to backport this to v1.2

**Review by alexmt — APPROVED** [review]

**Comment by alexec** [comment]
Back-ported to v1.2

**Pull request merged**